docs/content/guides/rows/row-sorting.md

```diff
@@ -297,153 +982,253 @@ import 'handsontable/dist/handsontable.full.min.css';
+        inStock: true,
+      },
+      {
+        brand: 'Zoomzone',
+        model: 'Cycling Cap',
+        price: 717,
+        sellDate: '03/05/2020',
+        sellTime: '88-2768',
+        inStock: false,
+      },
+      {
+        brand: 'Meeveo',
+        model: 'Full-Finger Gloves',
+        price: 314,
+        sellDate: '27/03/2020',
+        sellTime: '51-6775',
+        inStock: true,
+      },
+      {
+        brand: 'Buzzdog',
+        model: 'Mountain Frame',
+        price: 268,
+        sellDate: '29/08/2020',
+        sellTime: '44-4028',
+        inStock: false,
+      },
+      {
+        brand: 'Katz',
+        model: 'Half-Finger Gloves',
+        price: 204,
+        sellDate: '02/10/2020',
+        sellTime: '08-2758',
+        inStock: true,
+      },
+    ]}
+      columns={[
+        {
+          type: 'text',
+          data: 'brand',
+        },
+        {
+          type: 'text',
+          data: 'model',
+        },
+        {
+          type: 'numeric',
+          data: 'price',
+          numericFormat: {
+            pattern: '$ 0,0.00',
+            culture: 'en-US'
+          },
+        },
+        {
+          type: 'date',
+          data: 'sellDate',
+        },
+        {
+          type: 'text',
+          data: 'sellTime',
+        },
+        {
+          type: 'checkbox',
+          data: 'inStock',
+        },
       ]}
-      width="100%"
-      height={320}
-      colHeaders={['Brand', 'Model', 'Max speed<br>(in miles per hour)', 'Range<br>(in miles)', 'Seats', 'Price', 'Start of<br>production']}
-      columns={[{
-        data: 'brand' // 1st column is simple text, no special options here
-      }, {
-        data: 'model' // 2nd column is simple text, no special options here
-      }, {
-        data: 'maxSpeed',
-        type: 'numeric'
-      }, {
-        data: 'range',
-        type: 'numeric'
-      }, {
-        data: 'seats',
-        type: 'numeric'
-      }, {
-        data: 'price',
-        type: 'numeric',
-        numericFormat: {
-          pattern: '$ 0,0.00',
-          culture: 'en-US'
-        }
-      }, {
-        data: 'productionDate',
-        type: 'date',
-        dateFormat: 'MM/DD/YYYY',
-        correctFormat: true,
-        defaultDate: '01/01/1900'
-      }]}
-      columnSorting={true}
+      colHeaders={[
+        'Brand',
+        'Model',
+        'Price',
+        'Date',
+        'Time',
+        'In stock',
+      ]}
+      // enable sorting by multiple columns (for the entire grid)
+      multiColumnSorting={true}
+      rowHeaders={true}
+      height="auto"
+      width="auto"
       licenseKey="non-commercial-and-evaluation"
     />
   );
 };
 
 /* start:skip-in-preview */
-ReactDOM.render(<ExampleComponent />, document.getElementById('example3'));
+ReactDOM.render(<YourHandsontableComponent />, document.getElementById('example3'));
 /* end:skip-in-preview */
 ```
 :::
-:::
 
+:::
 
-## Custom compare functions
+## Set an initial sort order
```

Comment:
We need to extend this section a bit to make it more meaningful and complete. We can do this by adding a showcase in which we present the multi-column sorting by two columns (Brand and Model) in ascending order (Brand), and descending order (Model).

Avoid using column indices. Use names instead.

Reply:
Ok, how about this: [661cc00](https://github.com/handsontable/handsontable/pull/10183/commits/661cc0036c10ebec5e8e2ea583c85762a6fe58df)

Reply:
> Avoid using column indices. Use names instead.
Even in the comments section, I would write "sort column "Model" ascending" instead of "sort column 1", especially that "column 1" means "index 0" which might not be that obvious for some of the users.